tests/test_s3.py

```diff
@@ -1,12 +1,14 @@
+import json
+import re
 from http import HTTPStatus
 
 import pytest
 from botocore.exceptions import ClientError
 
 
-def test_s3_archive_file_in_bucket(mocked_s3, s3_client):
+def test_s3_client_archive_file_with_new_key_success(mocked_s3, s3_client):
```

Comment:
Good rename!

Reply:
I've been trying to keep up with my test naming conventions / structure since our work on the Transmog refactor! 🤓